portfolio/src/main/webapp/notes.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Notes</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+
```

Comment:
Whitespace outside `<pre>` elements may have unexpected consequences and should be avoided in HTML. Use CSS to add spaces.

Reply:
Just to double check, not the same thing as <code>\<pre></code> tag right?

Reply:
And also, when you say there shouldn't be whitespace, does it mean I shouldn't indent any of them?

Reply:
Feel free to indent. I suggest you use a tool like [htmltidy](http://www.html-tidy.org) to format HTML in a consistent way.

A `<pre></pre>` tag pair becomes a single `pre` element in the DOM.